CONTRIBUTING.md

```diff
@@ -165,6 +165,22 @@ If you do not have commit access, please ask one of the code owners to trigger t
 For more details on Swift-DocC's continuous integration, see the
 [Continous Integration](#continuous-integration) section below.
 
+### Introducing source breaking changes
+
+We try to avoid source breaking changes, for example:
+
+- removing or renaming public API
+- adding protocol requirements to public API
+- renaming Render JSON keys or adding new required Render JSON keys
+
+That said, sometimes there are good reasons to make these changes. In those cases we make an effort to offer
+a backwards compatible transition by deprecating the old API, alongside the new API, and keeping the deprecated
+API for one full minor release. For example, if we deprecate an API sometime after the 5.5 release we keep the
+deprecated API for the remainder of the upcoming release (5.6) and the entirety of the next release (5.7).   
+
+To indicate in-source when an was deprecated and when it we be removed, we add comments that say when we will
```

Comment:
```suggestion
To indicate in-source when an API was deprecated and when it will be removed, we add comments that say when we will
```

Reply:
Thanks. I missed these in my initial PR. 

Some people on the team talked about this offline and decided that 
 - we want this guidance to be a little more flexible
 - we want to indicate in the deprecation message when the specific API will be removed.
 
 I updated this paragraph to address those changes which also fixed these issues.